src/itree.cpp

```diff
@@ -31,13 +38,15 @@ auto print_dir(const fs::path&,
 auto correct_path(const po::variables_map&, rodsEnv&) -> fs::path;
 auto print_usage() -> void;
 auto get_json(const fs::path&, const po::variables_map&, ix::client_connection&, unsigned int depth) -> json::json;
+auto contains_pattern(const pattern_matcher& pm) -> bool;
+
```

Comment:
Extra line

Reply:
I took out the extra line.  However, a thing occurred to me: the <cassert> remains in 4-2-stable.  It won't hurt anything obvs., but these differences  that get introduced because of late-breaking review requests after one branch has already been merged seem ... problematic, if we desire 100% consistency.  

In terms of their effects in the code, of course, both of these branch differences are completely harmless. But I'll leave it up to others whether to apply the deletion of <cassert> and the blank line in the final product, and then also (obvs.) to whether to apply same by re-cherrypicking to 4-3-stable.

Reply:
You can include those last minute changes as a separate commit and then everything is aligned across branches.

Then all you have to do is cherry-pick the additional commit to the other branches.

Reply:
Seems good, it just won't have a prefixed issue number in the title of the(new)4-2- stable pull request. Ok

Reply:
Sure. As long as the additional commit has an issue number, it is fine.

Reply:
This has been addressed, so I will resolve this. Thanks